app/forms/date_form.py

```diff
@@ -16,191 +16,125 @@
     DateCheck,
     OptionalForm,
     DateRequired,
-    MonthYearCheck,
-    YearCheck,
     SingleDatePeriodCheck,
 )
 from app.utilities.schema import load_schema_from_metadata
 
 logger = logging.getLogger(__name__)
 
 
-class DateField(FormField):
-    def __init__(self, answer_store, metadata, answer, error_messages, **kwargs):
-        form_class = get_date_form(answer_store, metadata, answer, error_messages)
-        super(DateField, self).__init__(form_class, **kwargs)
-
-    def process(self, formdata, data=None):
-        if data is not None:
-            substrings = data.split('-')
-            data = {'year': substrings[0], 'month': substrings[1], 'day': substrings[2]}
-
-        super().process(formdata, data)
-
+class DateFormTypes(Enum):
```

Comment:
:+1:

Though, I think this should be called `DateFormType`. It represents a single type.